docsets/Jakarta_EE_10/docset.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "Jakarta EE 10",
+    "version": "10.0.0",
+    "archive": "Jakarta_EE_10.tgz",
```

Comment:
You need to include this archive at this path: `docsets/Jakarta_EE_10/Jakarta_EE_10.tgz`.

Reply:
fixed, and added specific versions for 10.0.0